docs/site/werf-docs-builder.inc.yaml

```diff
@@ -46,7 +46,7 @@ shell:
 {{ if ne .ModuleName "docs" }}
 ---
 artifact: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
```

Comment:
```suggestion
image: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
final: false
```

Reply:
This change is outside the scope of the common.
If I change artifact in this file, it will create a chain of changes in the whole file.